### ISSUE TITLE
Add other operating system to disable device from update-mgdevice command

### ISSUE THREAD

**Issue by freestyll** [microsoftgraph/msgraph-sdk-powershell#2723]
### Is your feature request related to a problem? Please describe the problem.

I try to disable a device with the command : update-mgdevice -deviceid "IDofTheDevice" -AccountEnabled:$false

But I get that answer when I tried for an Android or iOS device : Update-MgDevice_UpdateExpanded: Properties other than ExtendedAttribute1..15 can be modified only on windows devices.

For a windows device : success

### Describe the solution you'd like.

Add android and iOS capabilities for that update-device command, we should have the same capabilities between shell command and from azure portal.

**Comment by petrhollayms** [comment]
Hi @freestyll ,

Could you please execute the cmdlet in a debug mode (-debug) and share the log? 
Thanks!

**Comment by freestyll** [comment]
Hello, 
Yes I got that answer : 

`Update-MgDevice_Update: Properties other than ExtendedAttribute1..15 can be modified only on windows devices.`

Thank you

**Comment by petrhollayms** [comment]
Ok, thanks for sharing!

This appears to be an issue or limitation with the service APIs. Unfortunately, as the Microsoft Graph SDK team, we do not have ownership of the APIs that are causing you issues. We invite you to create a question about the service API to Microsoft Q&A and tagged with one of the [microsoft-graph-*] tags, that way it will get routed to the appropriate team for them to triage:  
 
https://aka.ms/msgraphsupport or directly https://aka.ms/askgraph

**Comment by freestyll** [comment]
Yes exactly, this is a limitation, sadly what we can do from Web GUI aren't the same with API. 
Following your advice, I will create a question about API to Microsoft Q&A. 
Thank you for your helps.
